keyboards/doio/kb38/info.json

```diff
@@ -22,8 +22,8 @@
     "processor": "atmega32u4",
     "usb": {
         "device_version": "1.0.0",
-        "pid": "0x0000",
-        "vid": "0xFEED"
+        "pid": "0x3801",
+        "vid": "0xD010"
```

Comment:
I'm confused by this change, I first remarked on this over on the [via repo](https://github.com/the-via/keyboards/pull/2224#discussion_r1578679264), but it seems like the kb38 I have reports a `vid` of `0xFEED` (matching the old value), and a pid of `0x6060` (which doesn't match either the old or new value for this, but _does_ match the `pid` in the `via` dev json [linked on the keebmonkey wiki](https://wiki.keebmonkey.com/en/products/mmp). Is this something there's a firmware update in the works to change or something?